### ISSUE TITLE
Importing Swagger 2.0 without 'summary' field produces broken Postman collection - no names in operations.

### ISSUE THREAD

**Issue by centur** [a85/POSTMan-Chrome-Extension#4458]
## App Details:
```
Postman for Windows
Version 6.0.10
win32 10.0.16299 / x64
```

## Issue Report:
**Did you encounter this recently, or has this bug always been there**: Recently, previously no such issue observed.

Issue : If Swagger definition doesn't contain `Summary` field for operation - Postman's collection entry will have empty "Name" and it won't be possible to edit it in an existing entry tab.

Steps to reproduce the problem:
1. Take this [Petstore sample.txt](https://github.com/postmanlabs/postman-app-support/files/1913713/Petstore.sample.txt), rename it to .json and import into Postman. 
**NOTE:** This swagger sample was intentionally modified - I removed `summary` property in one of the operations (OperationId: 'addPet') as it's not a mandatory field in Swagger 2.0 specification.

2. You will see a collection entry with no "Name" value, neither this value can be entered or edited if the entry is open:

![brokenentry](https://user-images.githubusercontent.com/720822/38787770-6c08d0c6-4173-11e8-894e-c41659eab6a4.png)

3. User can only use "Save As" on this entry and specify name for a new entry. 

Expected behaviour:  Would be nice if Postman would fall back to previous behaviour - if there is no summary, it can use "OperationId" or constructed URL or some other field that is mandatory ( I believe operationId??URL fallback is good enough)

It is working on my colleague's machine with older version of Postman app:
```
Postman for Windows
Version 5.5.3
win32 10.0.16299 / x64
```

**Comment by sdnts** [comment]
Hi @centur, thanks for reporting this, I was able to import this collection and see the empty name. However, I don't understand what you mean by `neither this value can be entered or edited if the entry is open`.
Can you try using the shortcut `Cmd/Ctrl+E` on the sidebar to rename this? There is also a dropdown option to rename. I was able to rename it using both these ways. Can you give them a shot too?

The fallback, though, looks like an issue, and I'm marking this as a bug for now.

**Comment by centur** [comment]
Thanks, I forgot about that option (Ctrl+E). It's working, also I can click on triple dots and rename from there.
What I meant is this: 
I expect to see empty "Name" control where I can input name when I'm on tab with request. Compare these 2 screenshots:

 Valid Summary (and valid name with that `|>` arrow control to expand it)
![image](https://user-images.githubusercontent.com/720822/38849570-0f314bd4-4250-11e8-951b-08386e647f3b.png)

No summary and even impossible to change it - control is not just empty - it's missing completely
![image](https://user-images.githubusercontent.com/720822/38849644-774c2112-4250-11e8-87dc-08aa37612200.png)

for me it's not very useful to rename it in sidebar, I have to open each element individually, because my exported API looks like this:
![image](https://user-images.githubusercontent.com/720822/38849718-f5411924-4250-11e8-8946-768ccfbeb7f0.png)

**Comment by vinayan3** [comment]
Would it be possible to use the `operationId`  if there is no summary? It's required, unique, and often describes the operation. In my case I have an auto-generated swagger sepc from our API codebase and haven't overridden each of the functions to include summaries.

**Comment by centur** [comment]
any update on this issue progress ?

**Comment by centur** [comment]
Is there any chance this regression will be fixed ?

**Comment by cedricraud** [comment]
I am facing the same issue.

Swagger snippet : 
```
"/v1/account/email": {
      "put": {
        "tags": [
          "Account"
        ],
        "summary": "",
        "responses": {
          "403": {
            "description": "Unauthorized"
          },
          "200": {
            "schema": {
              "$ref": "#/definitions/User"
            },
            "description": "Success"
          }
        },
        "parameters": [
          {
            "type": "string",
            "required": true,
            "name": "Authorization",
            "in": "header",
            "description": "Authorization token"
          },
          {
            "schema": "string",
            "required": true,
            "name": "email",
            "in": "body",
            "description": "Updated email"
          },
          {
            "schema": "string",
            "required": false,
            "name": "password",
            "in": "body",
            "description": "User password, for security check"
          }
        ],
        "operationId": "Backend.API.V1.AccountController.update_email",
        "description": "Update user email"
      }
    },
```

Result of the import: 

![postman-import](https://user-images.githubusercontent.com/868781/45078409-415c2e80-b0f0-11e8-8943-4fdd73487d0d.png)

**Comment by zjrunner** [comment]
FYI I have some pull requests going over to [swagger2-postman2](https://github.com/postmanlabs/swagger2-postman2/issues/19) as a workaround if the core doesn't get fixed.  I'm doing other swagger/postman merges and processing so needed the helper anyway.

**Comment by jdrews** [comment]
Seconded. This would be useful. Using operationId makes sense.

**Comment by abhijitkane** [comment]
@jdrews @cedricraud @centur This is fixed in the latest Postman version - 7.2+.
